plugins/chat/app/serializers/chat_channel_serializer.rb

```diff
@@ -93,6 +94,10 @@ def current_user_membership
     ).as_json
   end
 
+  def last_message
+    ChatMessage.where('chat_channel_id': object.id).last
```

Comment:
be sure you have ruby syntax tree configured correctly, this should be using double quotes